docs/partner_editable/data_format.adoc

```diff
@@ -179,16 +179,22 @@ The output returns a list of reading dates in which anomalies are marked on a da
 }
 ----
 
-==== Meter outage details
+==== Meter-outage details
+
+The endpoint returns a list of meters that delivered an error code instead of a valid reading. Data includes geographic information (latitude, longitude) which can be used to create a heat map of current meter outages, for example, to visualize an outage cluster.
+
+//TODO Shivansh, Is "heat map" correct above, or do we just mean "map"?
```

Comment:
heat map is correct.

Reply:
Sascha removed "heat" in a commit after this.